albumentations/augmentations/transforms.py

```diff
@@ -447,6 +448,49 @@ def get_transform_init_args_names(self):
         return ("height", "width", "interpolation")
 
 
+class Scale(DualTransform):
+    """Scale the input to the given scaling factor.
+
+    Args:
+        scale (float or (float, float)): scaling factor. If scale is a single float value -
+            input will be scaled proportionately. If tuple of floats - different scaling factors
+            will be used for the width and height of the input.
+        interpolation (OpenCV flag): flag that is used to specify the interpolation algorithm. Should be one of:
+            cv2.INTER_NEAREST, cv2.INTER_LINEAR, cv2.INTER_CUBIC, cv2.INTER_AREA, cv2.INTER_LANCZOS4.
+            Default: cv2.INTER_LINEAR.
+        p (float): probability of applying the transform. Default: 1.
+
+    Targets:
+        image, mask, bboxes, keypoints
+
+    Image types:
+        uint8, float32
+    """
+
+    def __init__(self, scale, interpolation=cv2.INTER_LINEAR, always_apply=False, p=1):
+        super(Scale, self).__init__(always_apply, p)
+
+        if isinstance(scale, (int, float)):
+            scale = scale, scale
+
+        self.scale = scale
+        self.interpolation = interpolation
+
+    def apply(self, img, interpolation=cv2.INTER_LINEAR, **params):
+        return F.scale(img, self.scale, interpolation=interpolation)
```

Comment:
This will not work for images with more than 4 channels. We should use a wrapper `maybe_process_in_chunks`. See ElasticTransform as a reference.

Reply:
`F.scale()` call `F.resize()` witch use `maybe_process_in_chunks`.